src/lib/localStorage.js

```diff
@@ -0,0 +1,11 @@
+export const jwtDecode = (t) => {
+  let token = {};
```

Comment:
You should probably check if `t` is null

Reply:
You should check to see if `t` is null.